redbot/core/commands/help.py

```diff
@@ -579,19 +579,20 @@ async def command_not_found(self, ctx, help_for, help_settings: HelpSettings):
             ret = await format_fuzzy_results(ctx, fuzzy_commands, embed=use_embeds)
             if use_embeds:
                 ret.set_author(
-                    name=f"{ctx.me.display_name} {T_('Help Menu')}", icon_url=ctx.me.avatar_url
+                    name=_("{ctx.me.display_name} 'Help Menu'").format(ctx=ctx),
```

Comment:
Quotes weren't part of the string her.
```suggestion
                    name=_("{ctx.me.display_name} Help Menu").format(ctx=ctx),
```